### DISCUSSION TITLE
Generic OpenID / OAuth2 Question

### DISCUSSION THREAD

**Discussion by vanillacandy** [requarks/wiki#3490] [Help / Questions]
**Question**
My SSO provider allows openid and profile email.read scope. I filled in all the necessary information in configuration

Here's my error output:
<img width="596" alt="Screen Shot 2020-11-19 at 10 34 26 AM" src="https://user-images.githubusercontent.com/6863496/99711313-7a291980-2a56-11eb-88f2-6b079a2a91ae.png">

Here's my SSO allowed scopes:
<img width="445" alt="Screen Shot 2020-11-19 at 10 50 49 AM" src="https://user-images.githubusercontent.com/6863496/99711324-7eedcd80-2a56-11eb-9525-e9ed068eb858.png">

<img width="723" alt="Screen Shot 2020-11-20 at 12 18 51 PM" src="https://user-images.githubusercontent.com/6863496/99846094-97c7b300-2b2a-11eb-8f3e-79c3a9739f23.png">

Here's the field I input "email.read"
<img width="218" alt="Screen Shot 2020-11-19 at 10 51 01 AM" src="https://user-images.githubusercontent.com/6863496/99711346-8319eb00-2a56-11eb-8dc4-6708ab43b3e5.png">

**Host Info (please complete the following information):**
OS: [e.g. Ubuntu 18.04, Docker]
Wiki.js version: 2.0.5
Database engine: [e.g. postgres 9.7, mysql 5.7]

**Comment by bagavan-i95dev** [comment]
How to pass the custom scopes in the Generic OpenID Connect / OAuth2 request, It always going as "openid openid profile email"
